tests/unit/core/data/datasets/test_segmentation.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Class definition for detection model entity used in OTX."""
+
+from __future__ import annotations
+
+import pytest
+from datumaro import Dataset as DmDataset
+from otx.core.data.dataset.segmentation import OTXSegmentationDataset
+from otx.core.data.entity.segmentation import SegDataEntity
+
+
+class TestOTXSegmentationDataset:
+    @pytest.fixture()
+    def dataset_generator(self) -> OTXSegmentationDataset:
```

Comment:
This seems covered by the existing test and fixture parametrization:
https://github.com/openvinotoolkit/training_extensions/blob/479868bdff52046f1e1b0c09395b7fbc13c26084/tests/unit/core/data/conftest.py#L63-L74
https://github.com/openvinotoolkit/training_extensions/blob/479868bdff52046f1e1b0c09395b7fbc13c26084/tests/unit/core/data/test_dataset.py#L11-L31
Do we need this still?